source/wp-content/themes/wporg-gutenberg/style-mobile.css

```diff
@@ -25,6 +25,18 @@ h3 {
 .wp-block-button__link {
 	border-radius: var(--wp--custom--button--border--radius);
 	background-color: #32373c;
+	background-color: var(--wp--custom--button--color--background);
```

Comment:
The styles added here (L28 - L39) are just for making the layout in the local env consistent with the sandbox and the prod. (i.e., the styles on the sandbox and the prod are already aligned)